lib/auth.ts

```diff
@@ -4,11 +4,16 @@ import dbConnect from '@/lib/dbConnect';
 import Salon from '../app/models/salon';
 import {stripe} from '@/lib/stripe';
 import {resolveControllerParams} from './utils';
+import Stripe from 'stripe';
 
 function getRandomInt(min: number, max: number) {
   return Math.floor(Math.random() * (max - min + 1)) + min;
 }
 
+function assertUnreachable(option: string) {
```

Comment:
This should be typed with `never`, so that the switch statement gives a compile time error when there is a new option